drivers/hmis/app/graphql/types/hmis_schema/external_form_submission.rb

```diff
@@ -0,0 +1,32 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Types
+  class HmisSchema::ExternalFormSubmission < Types::BaseObject
+    include Types::HmisSchema::HasCustomDataElements
+
+    available_filter_options do
+      arg :status, HmisSchema::Enums::ExternalFormSubmissionStatus
+      arg :submitted_date, GraphQL::Types::ISO8601Date
+    end
+
+    field :id, ID, null: false
+    field :submitted_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :spam_score, Float, null: true
+    field :status, HmisSchema::Enums::ExternalFormSubmissionStatus, null: false
+    field :notes, String, null: true
+    field :definition, Forms::FormDefinition, null: false
+    custom_data_elements_field
+
+    def custom_data_elements
+      resolve_custom_data_elements(object, definition_scope: definition.custom_data_element_definitions)
```

Comment:
Based on playing around with the UI changes, I think we should order these somehow so they come back in a consistent and sensible order (e.g. first name then last name). `.order(id: :asc)` maybe?

Reply:
Sounds good, it should sort by key now. Also adjusted the resolver to use a data loader.

Probably relying on id/key order isn't really right and the cdeds should have an optional sort order. But we can tackle that down the road